opensquirrel/circuit.py

```diff
@@ -64,9 +66,6 @@ def from_string(
             gate_set: an array of gate semantic functions. See default_gates for examples
             gate_aliases: a dictionary of extra gate aliases, mapping strings to functions in the gate set
             measurement_set: an array of measurement semantic functions. See default_measurements for examples
-            measurement_aliases: a dictionary of measure aliases, mapping strings to functions in the measurement set
```

Comment:
Is this correct? Removing `measurement_aliases` line?

Reply:
Yes, it does not exist anymore.